bindings/python/scripts/dlite-validate

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+"""Validator for dlite datamodels.
+"""
+import argparse
+import json
+import re
+import yaml
+from pathlib import Path
+
+import dlite
+
+
+def parse(url, driver, options=None):
+    """Parse url and return an instance."""
```

Comment:
Can we have a little longer docstring that also includes some examples of usage for the various options?

Reply:
Improved docstring has now been added together with --id and --storage-path options for validating data instances.